modules/technology-preview.adoc

```diff
@@ -0,0 +1,16 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+{FeatureName} is a Technology Preview release only. Technology Preview releases
+are not supported with Red Hat production service-level agreements (SLAs) and
+might not be functionally complete, and Red Hat does not recommend using them
+for production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process. For more information see
+link:https://access.redhat.com/support/offerings/techpreview/ [Red Hat
```

Comment:
Need to remove the space here for the link to function correctly.

Reply:
Whoops! Fixed. Thanks!